website/content/docs/configuration/storage/raft.mdx

```diff
@@ -115,10 +115,11 @@ delay) mode. The maximum allowed value is 10.
   in the Raft quorum but will still receive the data replication stream, adding
   read scalability to a cluster. This option has the same effect as the
   [`-non-voter`](/vault/docs/commands/operator/raft#non-voter) flag for the
-  `vault operator raft join` command, but only affects voting status when joining
-  via `retry_join` config. This setting can be overridden to true by setting the
-  `VAULT_RAFT_RETRY_JOIN_AS_NON_VOTER` environment variable to any non-empty value.
-  Only valid if there is at least one `retry_join` stanza.
+  `vault operator raft join` command, but only affects voting status when
+  joining via `retry_join` config. This setting can be overridden to true by
+  setting the `VAULT_RAFT_RETRY_JOIN_AS_NON_VOTER` environment variable to any
+  non-empty value. Only valid if there is at least one `retry_join` stanza. Use
+  of this feature requires Vault Enterprise.
```

Comment:
```suggestion
  joining via `retry_join` config. You force the retry setting to be `true` by
  setting the `VAULT_RAFT_RETRY_JOIN_AS_NON_VOTER` environment
  variable to any non-empty value. Retrying is only valid if there is at least
  one `retry_join` stanza. <EnterpriseAlert inline />
```
Style correction: write in active voice, avoid "this" as a pronoun

Reply:
Thanks Sarah! I'm terrible at recognising active/passive voice. I will change these cases as you've suggested.

One question though is around consistency. I generally took the lead on phrasing from all the other options on this page. In face this specific change wasn't my writing - I only added the missing `Use of this feature requires Vault Enterprise.` because we failed to document that it's an ent-only feature at all.

Do you prefer new edits to change the voice even when the rest of this page remains the same? I'd prefer not to rewrite the entire page just to add docs for this one new parameter so would love your thoughts on active voice vs. consistency on the page.

Actually the Enterprise marker I added is another consistency issue: the others on this page simply included the text inline (though I was expecting something more like the `EnterpriseAlert` you proposed). Do we have a document anywhere on what the "correct" format is? Should I go and change all the other instances on this page too to keep them consistent?

Reply:
> Do you prefer new edits to change the voice even when the rest of this page remains the same? 

Yes, because we'd never improve the documentation otherwise. It's better to make incremental improvements even if it makes things a bit inconsistent. The chances that we will have the time to holistically re-write/update the whole document as part of routine maintenance is almost zero, because we don't have a big enough writing team.

> Do we have a document anywhere on what the "correct" format is?

Unfortunately not. It's a known hole that I'm trying to fix in the near-to-mid term :(

> Should I go and change all the other instances on this page too to keep them consistent?

If you have the bandwidth to do so, that would be incredibly helpful :)

Reply:
Thanks Sarah. I did update those other entries too in the end. Let me know if I addressed your concerns!